docs/cmd/kn_service_update.md

```diff
@@ -41,6 +41,11 @@ kn service update NAME [flags]
   -p, --port int32               The port where application listens on.
       --requests-cpu string      The requested CPU (e.g., 250m).
       --requests-memory string   The requested memory (e.g., 64Mi).
+      --tag-latest string        Tag latest ready revision, format: --tag-latest tag , example: --tag-latest current
+      --tag-revision strings     Tag revisions, format: --tag-revision revision:tag , example: --tag-revision echo-abcde:current (can be specified multiple times)
```

Comment:
I assume these `—tag-revision` and `—untag-revision` will match the `—label` work being done in #328? If yes, then we need to agree on one name IMO. Different names for same thing will only confuse user.

Reply:
#328 is mostly unrelated to this feature. #328 is more like `kn service create --env` which we already have. 

However, instead of `--untag-revision current`  we coluld also use `--tag-revision :current` (i.e. keep the revision part empty). Not sure if this looks much cleaner though (especiall if we might consider to switch to `=` as separator then this would mutate to `--tag-revision =current` or even `--tag-revision==current`.

Normally I'm always for introducing as less options as possible, but in this case, I think a dedicated `--untag-revision` is justified.

Another question is around whether we want to have `--tag-revision` or whether its not already clear from the context (tags can be only on revisions), that we could reduce that to `--tag` and `--untag`

Reply:
I'd vote for `--tag` `--untag` since context is in the group name.